src/utils.ts

```diff
@@ -121,6 +121,10 @@ export namespace ControllerUtils {
     return res.status(503).send({ message });
   };
 
+  export const unableToProcess = (res: Response, message: string): any => {
+    return res.status(422).send({ message });
```

Comment:
Don't think I've ever seen a 422 in the wild, is kinda cool. According to what I read though 422 implies the user semantics are correct (ie. correctly formed file). I think we just want a 403 response here.